src/main/groovy/io/cify/runner/utils/CapabilityParser.groovy

```diff
@@ -49,17 +62,37 @@ class CapabilityParser {
     }
```

Comment:
Could you please explicit return statement?

Reply:
Yes, added return statements to all methods